libs/domain-service/src/resolvers/index.ts

```diff
@@ -8,6 +9,8 @@ import {
   getRegistriesForDomain,
 } from "../registries";
 
+allSettled.shim(); // Makes it compatible with React Native when Promise.allSettled isn't avaiblable
```

Comment:
this is not to be done in the lib but in React Native side then.

Reply:
This lib will be used externally, so no it should be there as well to make sure any external wallet can use it independently of the platform.

Reply:
a library must not have a global shim side effect. it is either on user land to fix OR just don't use allSettled in this library.

Reply:
it's for the end projects to deal with this, typically if we upgrade React Native we may not need it anymore and therefore it kinda belongs to the polyfills the same way we didn't polyfill live-common "in place" just because React Native needed tons of polyfill to make it run.

Reply:
Done, shim moved to LLM + Error thrown in domain-service lib